statusboard/src/utils/utils.ts

```diff
@@ -104,9 +104,7 @@ export function filterProjectsByTime(
 ) {
   const timeRanges = timeRangeKey ? ACTIVE_THRESHOLDS[timeRangeKey] : undefined;
   if (!timeRanges) return projects;
-  return projects.filter((p) =>
-    p.last_pushed_within ? timeRanges.includes(p.last_pushed_within) : false
-  );
+  return projects.filter((p) => timeRanges.includes(p.last_pushed_within));
```

Comment:
The changes here are necessary because 

`ACTIVE_THRESHOLDS[timeRangeKey]` is of type `(string | undefined)[]`.

and in

https://github.com/codeforamerica/brigade-project-index-statusboard/blob/a7069c8122f5feb98e9dac336dfdfc969c5e3118/statusboard/src/utils/utils.ts#L6-L14

it seems like `all time` allows the value to be `undefined`. (i.e., if you want to match all projects, include projects whose `last_pushed_within` is undefined)

I don't know the logic why `last_pushed_within` can be undefined :/

Reply:
Projects can exist without references to any git repository, or their git repository can be unresolvable, and in those cases `last_pushed_within` can be undefined